db/migrate/20150504185433_add_workflow_version_to_classification.rb

```diff
@@ -0,0 +1,7 @@
+class AddWorkflowVersionToClassification < ActiveRecord::Migration
+  def change
+    add_column :classifications, :workflow_version, :text
+    Classification.update_all("workflow_version = metadata->>'workflow_version'")
```

Comment:
Should we remove the metadata[:workflow_version] field as well to have one source of this data?